server/routes/posts.js

```diff
@@ -186,4 +186,77 @@ router.put('/:postID/likes/:action', async (req, res, next) => {
     }
 })
 
+// POST /posts/comments
+// Retrieves all comments corresponding to the comment IDs provided
+router.post('/comments', async (req, res, next) => {
```

Comment:
```suggestion
router.post('/comments', async (req, res, _next) => {
```

unused vars here can be preceded by a _ to show that they're unused

Reply:
Wow I had no idea that was a thing, cool

[mark next as unused](https://github.com/Zen-L-MU2025/project-sk8rlog/commit/80a9ff2d86f3fb6f7642a667d0dbaab591ed9aae)

Reply:
it's just a naming convention! doesn't really actually do anything haha. Some linters will have you add the underscore.